docs/Doxyfile.in

```diff
@@ -814,10 +814,10 @@ WARN_LOGFILE           =
 # spaces. See also FILE_PATTERNS and EXTENSION_MAPPING
 # Note: If this tag is empty the current directory is searched.
 
-INPUT                  = @PROJECT_SOURCE_DIR@/src
-INPUT                 += @PROJECT_SOURCE_DIR@/test
-INPUT                 += @PROJECT_BINARY_DIR@/src/ast
-INPUT                 += @PROJECT_BINARY_DIR@/src/visitors
+INPUT                  = @NMODL_PROJECT_SOURCE_DIR@/src
+INPUT                 += @NMODL_PROJECT_SOURCE_DIR@/test
+INPUT                 += @CMAKE_BINARY_DIR@/src/ast
+INPUT                 += @CMAKE_BINARY_DIR@/src/visitors
```

Comment:
Initially we were mixing PROJECT_BINARY_DIR and CMAKE_BINARY_DIR.

If you remmeber our conversation last friday, PROJECT_BINARY_DIR is binary directory of last `project` command in CMakeLists.txt. So in case of submodule, there will be new `project` command and hence there will be new directory `<build>/external/nmodl`.

To "simplify", I thought to simply use `CMAKE_BINARY_DIR` everywhere so that all stuff can be found in `<build>` directory itself (as I did for coreneuron + neuron integration).

Reviewing this choice, I now think using `PROJECT_BINARY_DIR` everywhere make more sense to isolate the build scope and not pollute top level project's build directory.

I have done now necessary changes.

Reply:
Yes that was my thought too. I agree with your new changes